pages/common/jekyll.md

```diff
@@ -0,0 +1,15 @@
+# Jekyll
+
+> Jekyll is a simple, blog-aware, static site generator.
+
+- Generate a development server that will run at http://localhost:4000/:
+
+`jekyll serve`
+
+- Enable incremental regeneration:
+
+`jekyll serve --incremental`
+
+- Generated the current folder into ./_site:
```

Comment:
Perhaps `Generated` -> `Generate`? Also, backticks around `./_site` would help neaten things up :-)

Reply:
Thank you for the input. I've corrected my PR.